presentation-core/src/main/java/tachiyomi/presentation/core/components/WheelPicker.kt

```diff
@@ -149,23 +149,29 @@ private fun <T> WheelPicker(
             }
 
             val scope = rememberCoroutineScope()
+            val processManualInput: () -> Unit = {
+                scope.launch {
+                    items
+                        .indexOfFirst { it.toString() == value.text }
+                        .takeIf { it >= 0 }
+                        ?.apply {
+                            internalOnSelectionChanged(this)
+                            lazyListState.scrollToItem(this)
+                        }
+                    showManualInput = false
+                }
+            }
+
             BasicTextField(
                 modifier = Modifier
                     .align(Alignment.Center)
                     .showSoftKeyboard(true)
                     .clearFocusOnSoftKeyboardHide {
-                        scope.launch {
-                            items
-                                .indexOfFirst { it.toString() == value.text }
-                                .takeIf { it >= 0 }
-                                ?.apply {
-                                    internalOnSelectionChanged(this)
-                                    lazyListState.scrollToItem(this)
-                                }
-
-                            showManualInput = false
-                        }
+                        processManualInput()
                     },
+                onKeyboardAction = {
+                    processManualInput()
+                },
```

Comment:
how about this? same thing above
```suggestion
                onKeyboardAction = processManualInput,
```